test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
+
+describe('A hash map of texture info, similar to TexturePacker', function () {
```

Comment:
Ok, I'll call it 'An object map of texture info, similar to TexturePacker'? What I'm referring to is what it is outputting, I don't really understand what `configUtils.multipleSprites` has to do with it?